tests/lax_scipy_test.py

```diff
@@ -125,13 +126,19 @@ def lax_fun(array_to_reduce):
   def testScipySpecialFun(self, scipy_op, lax_op, rng_factory, shapes, dtypes,
                           test_autodiff):
     rng = rng_factory(self.rng())
+    if scipy_op is osp_special.polygamma:
+      dtypes = list(dtypes)
+      dtypes[0] = onp.int32 if dtypes[0] == onp.float32 else onp.int64
     args_maker = self._GetArgsMaker(rng, shapes, dtypes)
     args = args_maker()
     self.assertAllClose(scipy_op(*args), lax_op(*args), atol=1e-3, rtol=1e-3,
                         check_dtypes=False)
     self._CompileAndCheck(lax_op, args_maker, rtol=1e-5)
 
     if test_autodiff:
+      if scipy_op is osp_special.polygamma:
+        lax_op = partial(lax_op, args[0])
+        args = args[1:]
```

Comment:
Same here... perhaps add an optional `static_argnums` field to `rec` to handle this case and avoid hard-coding `polygamma` in the test infrastructure.

Reply:
Thanks, I have added the option `nondiff_argnums` (as in `custom_jvp`) for that purpose.